docs/topics/ksp/ksp-examples.md

```diff
@@ -3,15 +3,15 @@
 ## Get all member functions
 
 ```kotlin
-fun KSClassDeclaration.getDeclaredFunctions(): List<KSFunctionDeclaration> =
+fun KSClassDeclaration.getDeclaredFunctions(): Sequence<KSFunctionDeclaration> =
     declarations.filterIsInstance<KSFunctionDeclaration>()
 ```
 
 ## Check whether a class or function is local
 
 ```kotlin
 fun KSDeclaration.isLocal(): Boolean =
-    parentDeclaration != null && parentDeclaration !is KSClassDeclaration
+    parentDeclaration is KSClassDeclaration
```

Comment:
This is not correct. `isLocal()` means if a declaration is within a function body, as per the above comment says, therefore there is no need to fix this.